expert/ezproxy.cfg

```
@@ -513,6 +513,18 @@ DJ future-science.com
 # Gale
 IncludeFile databases/gale.txt
 
+# Haaretz English
+# https://phabricator.wikimedia.org/T355668
+Title Haaretz English
+URL https://www.haaretz.com/
+HJ https://www.haaretz.com/
```

This is just a shotgun approach to try to rewrite more javascript.
```suggestion
HJ https://www.haaretz.com/
DJ haaretz.com
```